pkgs/top-level/aliases.nix

```diff
@@ -1289,6 +1289,7 @@ mapAliases ({
   tokodon = plasma5Packages.tokodon;
   tokyo-night-gtk = tokyonight-gtk-theme; # Added 2024-01-28
   tootle = throw "'tootle' has been removed as it is not maintained upstream. Consider using 'tuba' instead"; # Added 2024-02-11
+  tomcat_connectors = throw "'tomcat_connectors' has been renamed to 'apacheHttpdPackages.mod_jk'"; # Added 2024-03-21
```

Comment:
```suggestion
  tomcat_connectors = apacheHttpdPackages.mod_jk; # Added 2024-03-21
```

Reply:
Thanks for the review! Changed the alias to not to throw.